src/lib/selections.ts

```diff
@@ -146,6 +147,15 @@ export async function getEventForSelectWithPoint({
       engineCommandManager.artifactGraph
     )
     if (err(codeRef)) return null
+    if (_artifact?.subType === 'adjacent') {
+      return {
+        type: 'Set selection',
+        data: {
+          selectionType: 'singleCodeCursor',
+          selection: { range: codeRef.range, type: 'adjacent-edge' },
```

Comment:
Hopefull I'll get [the selections refactor](https://docs.google.com/document/d/16EROjbvGZnBg7Y43yaA36mS4SkC8Cxpm2gpZ1mY2UJo/edit#heading=h.2x5g017rxgoj) done soon, but in the mean time, there kinda needs to be a type that matches with the artifact types (not every case but a lot of them, in this case can't differentiate between an opposite edge vers adjacent with only the source range)